specification/datafactory/resource-manager/Microsoft.DataFactory/preview/2017-09-01-preview/entityTypes/Pipeline.json

```diff
@@ -1206,7 +1207,7 @@
       "properties": {
         "writeBehavior": {
           "description": "The write behavior for the operation. Default is 'Insert'.",
-          "type": "string",
+          "type": "object",
```

Comment:
Like the above, does this need to be an object if allowed values are all strings?